src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorDiagnosticsTests.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Text.Json.SourceGeneration.UnitTests
     [ActiveIssue("https://github.com/dotnet/runtime/issues/58226", TestPlatforms.Browser)]
     [SkipOnCoreClr("https://github.com/dotnet/runtime/issues/71962", ~RuntimeConfiguration.Release)]
     [SkipOnMono("https://github.com/dotnet/runtime/issues/92467")]
```

Comment:
I think we can remove the Mono skip and possibly also the CoreCLR one since according to https://github.com/dotnet/runtime/issues/71962 the issues we saw with Checked config also seemed to happen on x86 only.